### DISCUSSION TITLE
The webpage at http://localhost:3000/api/auth/callback/google....  might be temporarily down or it may have moved permanently to a new web address.

### DISCUSSION THREAD

**Discussion by Edartruwu** [nextauthjs/next-auth#11482] [Help]
For some reason I can't figure out, when I set up Google OAuth, I run into this issue:

![image](https://github.com/user-attachments/assets/82e02604-b50d-4ca5-b925-dcb55ebc1373)

### Context:

I'm using the Prisma adapter to implement Google OAuth in my system:

**`src/lib/auth.ts`**
```typescript
import NextAuth from "next-auth";
import { PrismaAdapter } from "@auth/prisma-adapter";
import { Client } from "@/utils/prisma";
import Google from "next-auth/providers/google";

export const { handlers, auth, signIn, signOut } = NextAuth({
  adapter: PrismaAdapter(Client),
  providers: [
    Google({
      clientId: process.env.AUTH_GOOGLE_ID,
      clientSecret: process.env.AUTH_GOOGLE_SECRET,
    }),
  ],
});
```

**`src/app/api/auth/[...nextauth]/route.ts`**
```typescript
import { handlers } from "@/lib/auth";
export const { GET, POST } = handlers;
```

I'm sure the `.env` file is set up correctly, and I've verified permissions in the Google console. I've also added myself as a test user in the Google console. When I click the sign-in button on my page, it redirects me to the Google account selector. However, after selecting my account, I encounter the error shown above. 😢

Any insights on what might be going wrong would be greatly appreciated!

**Comment by TimothyMendezFeliciano** [comment]
Did you ever resolve this??
I'm having the same issue with the same configuration.

**Reply by Edartruwu** [reply]
Yea i implemented my own auth using [arcticjs](https://arcticjs.dev/) and typescript, but its up to you lol ive seen a lot of people moving to [better-auth](https://www.better-auth.com/) too.
